Gruntfile.js

```diff
@@ -34,154 +34,135 @@ module.exports = function (grunt) {
+                    '<%= config.srcScriptsDir %>/**/*.js',
+                    '<%= config.compiledScriptsDir %>/**/templates.js',
                 ],
                 tasks: [
                     'compilejs',
-                    'uglify'
-                ]
+                    'uglify',
+                ],
             },
             swig: {
                 files: [
-                    '<%= config.templates.sourceFiles %>'
+                    '<%= config.templates.sourceFiles %>',
                 ],
                 tasks: [
-                    'orderedSwig'
-                ]
+                    'orderedSwig',
+                ],
             },
             ngtemplates: {
                 files: ['<%= config.templates.generatedFiles %>'],
-                tasks: ['ngtemplates']
-            }
-        },
-        browserify: {
-            all: {
-                files: {
-                    '<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'
-                }
-            }
+                tasks: ['ngtemplates'],
+            },
         },
+        browserify: {all: {files: {'<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'} } },
         babel: {
             all: {
-                options: {
-                    presets: ['es2015']
-                },
+                options: {presets: ['es2015']},
                 files: [{
                     expand: true,
                     cwd: '<%= config.srcScriptsDir %>',
                     src: ['**/*.es.js'],
                     dest: '<%= config.compiledScriptsDir %>',
-                    ext: '.js'
-                }]
-            }
+                    ext: '.js',
+                }],
+            },
         },
         eslint: {
+            options: {fix: grunt.option('fix-eslint')},
             node: {
-                options: {
-                    configFile: '.eslintrc.node'
-                },
-                src: '<%= config.lintFiles.node %>'
+                options: {configFile: '.eslintrc.node'},
+                src: '<%= config.lintFiles.node %>',
             },
             es: {
-                options: {
-                    configFile: '.eslintrc.es'
-                },
-                src: '<%= config.lintFiles.es %>'
+                options: {configFile: '.eslintrc.es'},
+                src: '<%= config.lintFiles.es %>',
             },
             tests: {
-                options: {
-                    configFile: '.eslintrc.es'
-                },
-                src: '<%= config.lintFiles.tests %>'
-            }
-        },
-        jscs: {
-            all: {
-                files: ['<%= config.lintFiles %>']
-            }
+                options: {configFile: '.eslintrc.es'},
+                src: '<%= config.lintFiles.tests %>',
+            },
         },
+        jscs: {all: {files: ['<%= config.lintFiles %>']} },
```

Comment:
As above, I think Grunt config should be laid out like JSON: `requirePaddingNewLinesInObjects: true` in JSCS. You could enable it at the top of the file.